### ISSUE TITLE
[BUG] Version 2.3.0: Commands and TAB-intentation don't work anymore ...

### ISSUE THREAD

**Issue by avirosso** [vslinko/obsidian-outliner#325]
The commands "fold list" & "unfold list" and the indentation "TAB" & "SHIFT-TAB" don't work anymore.

Nothing happens (so the plugin is useless)

All was fine with Outliner 2.2.5 ...

- OS: Windows 10
- Obsidian Version: 0.14.15 (Installer 0.14.15 too)
- Outliner Version 2.3.0 (I restarted Obsidian after updating Outliner)

**Comment by avirosso** [comment]
Maybe the problems are caused by changes for Obsidian 0.15.0 (CodeMirror 6). If so, the manifest.json should be updated too: The "minAppVersion": "0.13.23" should be changes to "minAppVersion": "0.15.0" ...

**Comment by mishamsk** [comment]
Same for me. Getting errors like this in the console:
```
TypeError: language.foldedRanges is not a function
    at MyEditor.getAllFoldedLines (eval at <anonymous> (app.js:1), <anonymous>:1022:28)
    at ParserService.parseWithLimits (eval at <anonymous> (app.js:1), <anonymous>:2293:36)
    at ParserService.parse (eval at <anonymous> (app.js:1), <anonymous>:2217:21)
    at PerformOperationService.performOperation (eval at <anonymous> (app.js:1), <anonymous>:2385:34)
    at SelectionShouldIgnoreBulletsFeature.run (eval at <anonymous> (app.js:1), <anonymous>:1831:42)
    at eval (eval at <anonymous> (app.js:1), <anonymous>:2176:61)
    at c (app.js:1)
    at pc (app.js:1)
    at Object.keydown (app.js:1)
    at e.runCustomHandlers (app.js:1)
```

**Comment by neekubee** [comment]
Same for me

**Comment by Archie-2021** [comment]
same for me, also i can't indent outdent with this plugin hot keys, so i am not sure how much is working, but the plugin as a whole is broken now

**Comment by kofm** [comment]
Same for "Draw vertical indentation lines". Works only if disabled

**Comment by mustafaalbazy** [comment]
Same here, the plugin completely stopped working for me in Obsidian 0.14.15 with latest Outliner update 2.3.0. 
Doesn't outline when pressing _Enter_ and no shortcuts working.

**Comment by mustafaalbazy** [comment]
> Same here, the plugin completely stopped working for me in Obsidian 0.14.15 with latest Outliner update 2.3.0.
> Doesn't outline when pressing _Enter_ and no shortcuts working.

Ok I solved the issue by downgrading to version 2.2.5 manually (just replace main.js, manifest.json, styles.css).

**Comment by vslinko** [comment]
Thank you all for the feedback. Sorry for the inconvenience.

I fixed the minAppVersion in the manifest.json file. Now you can't update to 2.3.0 without Obsidian 0.15.

But if you have already updated, you can copy `main.js` `styles.css` and `manifest.json` from [2.2.5](https://github.com/vslinko/obsidian-outliner/releases/tag/2.2.5) to the `.obsidian/plugins/obsidian-outliner` folder in your vault.
